File: SEPTEMBER-2025/30-09-2025/p2221.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        if(len(nums) == 1):
            return nums[0]
        else:
            newNum = []
            for i in range(len(nums)-1):
                newNum.append((nums[i] + nums[i+1]) % 10)
            return self.triangularSum(newNum)

File: SEPTEMBER-2025/30-09-2025/p2221.java
class Solution {
    public int triangularSum(int[] nums) {
        int n = nums.length;
        if(n == 0 || n == 1) return nums[0];

        for(int step = 1; step < n; step++){
            for(int i = 0; i < n - step; i++){
                nums[i] = (nums[i] + nums[i+1]) % 10;
            }
        }
        return nums[0];
    }
}